sat-plugin/src/main/resources/rulesets/checkstyle/rules.xml

```diff
@@ -201,8 +201,8 @@
     </module>
     <module name="org.openhab.tools.analysis.checkstyle.ForbiddenPackageUsageCheck">
       <property name="severity" value="warning" />
-      <property name="forbiddenPackages" value="${checkstyle.forbiddenPackageUsageCheck.forbiddenPackages}"/>
-      <property name="exceptions" value="${checkstyle.forbiddenPackageUsageCheck.exceptions}"/>
+      <property name="forbiddenPackages" value="${checkstyle.forbiddenPackageUsageCheck.forbiddenPackages}" default=""/>
```

Comment:
I don't know this syntax, but if you cannot do:

```
${checkstyle.forbiddenPackageUsageCheck.forbiddenPackages:""}
```

And this works then LGTM

Reply:
No, that didn't work. It still results in 
>Property ${checkstyle.forbiddenPackageUsageCheck.forbiddenPackages} has not been set

exception when building eSH.